pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -89,6 +89,17 @@ const (
 	// StateDeleting means we are in the process of cleaning up the host
 	// ready for deletion
 	StateDeleting ProvisioningState = "deleting"
+
+	// StateSoftPowerOffTriggered means host if soft powered off
+	// but not yet finished
+	StateSoftPowerOffTriggered ProvisioningState = "soft power off triggered"
+
+	// StateSoftPowerOffUnsupported means soft powered off command
+	// is not supported by the driver
+	StateSoftPowerOffUnsupported ProvisioningState = "soft power off unsupported"
```

Comment:
I don't think we need this state at all, do we? If we receive a power off request and the driver does not support soft power off we should just use the hard power off API.